# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# ブラシックデービッド, 2019
# Takefumi Nagata, 2019
# UTUMI Hirosi <utuhiro78@yahoo.co.jp>, 2020
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-12-07 17:09+0100\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: UTUMI Hirosi <utuhiro78@yahoo.co.jp>, 2020\n"
"Language-Team: Japanese (https://www.transifex.com/calamares/teams/20061/ja/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ja\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: src/modules/grubcfg/main.py:28
msgid "Configure GRUB."
msgstr "GRUBを設定にします。"

#: src/modules/mount/main.py:30
msgid "Mounting partitions."
msgstr "パーティションのマウント。"

#: src/modules/mount/main.py:127 src/modules/initcpiocfg/main.py:198
#: src/modules/initcpiocfg/main.py:202
#: src/modules/luksopenswaphookcfg/main.py:86
#: src/modules/luksopenswaphookcfg/main.py:90 src/modules/rawfs/main.py:164
#: src/modules/initramfscfg/main.py:85 src/modules/initramfscfg/main.py:89
#: src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/fstab/main.py:361
#: src/modules/fstab/main.py:367 src/modules/localecfg/main.py:135
#: src/modules/networkcfg/main.py:39
msgid "Configuration Error"
msgstr "コンフィグレーションエラー"

#: src/modules/mount/main.py:128 src/modules/initcpiocfg/main.py:199
#: src/modules/luksopenswaphookcfg/main.py:87 src/modules/rawfs/main.py:165
#: src/modules/initramfscfg/main.py:86 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/fstab/main.py:362
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "<pre>{!s}</pre> に使用するパーティションが定義されていません。"

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd services"
msgstr "systemdサービスを設定"

#: src/modules/services-systemd/main.py:59
#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "サービスが変更できません"

#: src/modules/services-systemd/main.py:60
msgid ""
"<code>systemctl {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"chroot で <code>systemctl {arg!s}</code> を呼び出すと、エラーコード {num!s} が返されました。"

#: src/modules/services-systemd/main.py:63
#: src/modules/services-systemd/main.py:67
msgid "Cannot enable systemd service <code>{name!s}</code>."
msgstr "<code>{name!s}</code>というsystemdサービスが可能にすることができません"

#: src/modules/services-systemd/main.py:65
msgid "Cannot enable systemd target <code>{name!s}</code>."
msgstr "systemd でターゲット <code>{name!s}</code>が開始できません。"

#: src/modules/services-systemd/main.py:69
msgid "Cannot disable systemd target <code>{name!s}</code>."
msgstr "systemd でターゲット <code>{name!s}</code>が停止できません。"

#: src/modules/services-systemd/main.py:71
msgid "Cannot mask systemd unit <code>{name!s}</code>."
msgstr "systemd ユニット <code>{name!s}</code> をマスクできません。"

#: src/modules/services-systemd/main.py:73
msgid ""
"Unknown systemd commands <code>{command!s}</code> and "
"<code>{suffix!s}</code> for unit {name!s}."
msgstr ""
"ユニット {name!s} に対する未知の systemd コマンド <code>{command!s}</code> と "
"<code>{suffix!s}</code>。"

#: src/modules/umount/main.py:31
msgid "Unmount file systems."
msgstr "ファイルシステムをアンマウント。"

#: src/modules/unpackfs/main.py:35
msgid "Filling up filesystems."
msgstr "ファイルシステムに書き込んでいます。"

#: src/modules/unpackfs/main.py:255
msgid "rsync failed with error code {}."
msgstr "エラーコード {} によりrsyncを失敗。"

#: src/modules/unpackfs/main.py:300
msgid "Unpacking image {}/{}, file {}/{}"
msgstr "イメージ {}/{}, ファイル {}/{} を解凍しています"

#: src/modules/unpackfs/main.py:315
msgid "Starting to unpack {}"
msgstr "{} の解凍を開始しています"

#: src/modules/unpackfs/main.py:324 src/modules/unpackfs/main.py:464
msgid "Failed to unpack image \"{}\""
msgstr "イメージ  \"{}\" の展開に失敗"

#: src/modules/unpackfs/main.py:431
msgid "No mount point for root partition"
msgstr "ルートパーティションのためのマウントポイントがありません"

#: src/modules/unpackfs/main.py:432
msgid "globalstorage does not contain a \"rootMountPoint\" key, doing nothing"
msgstr "globalstorage に \"rootMountPoint\" キーが含まれていません。何もしません。"

#: src/modules/unpackfs/main.py:437
msgid "Bad mount point for root partition"
msgstr "ルートパーティションのためのマウントポイントが不正です"

#: src/modules/unpackfs/main.py:438
msgid "rootMountPoint is \"{}\", which does not exist, doing nothing"
msgstr "ルートマウントポイントは \"{}\" ですが、存在しません。何もできません。"

#: src/modules/unpackfs/main.py:454 src/modules/unpackfs/main.py:458
#: src/modules/unpackfs/main.py:478
msgid "Bad unsquash configuration"
msgstr "unsquash の設定が不正です"

#: src/modules/unpackfs/main.py:455
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr "\"{}\" ({}) のファイルシステムは、現在のカーネルではサポートされていません"

#: src/modules/unpackfs/main.py:459
msgid "The source filesystem \"{}\" does not exist"
msgstr "ソースファイルシステム  \"{}\" は存在しません"

#: src/modules/unpackfs/main.py:465
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed"
msgstr "unsquashfs が見つかりませんでした。 squashfs-toolsがインストールされているか、確認してください。"

#: src/modules/unpackfs/main.py:479
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "ターゲットシステムの宛先 \"{}\" はディレクトリではありません"

#: src/modules/displaymanager/main.py:514
msgid "Cannot write KDM configuration file"
msgstr "KDMの設定ファイルに書き込みができません"

#: src/modules/displaymanager/main.py:515
msgid "KDM config file {!s} does not exist"
msgstr "KDM 設定ファイル {!s} が存在しません"

#: src/modules/displaymanager/main.py:576
msgid "Cannot write LXDM configuration file"
msgstr "LXDMの設定ファイルに書き込みができません"

#: src/modules/displaymanager/main.py:577
msgid "LXDM config file {!s} does not exist"
msgstr "LXDM 設定ファイル {!s} が存在しません"

#: src/modules/displaymanager/main.py:660
msgid "Cannot write LightDM configuration file"
msgstr "LightDMの設定ファイルに書き込みができません"

#: src/modules/displaymanager/main.py:661
msgid "LightDM config file {!s} does not exist"
msgstr "LightDM 設定ファイル {!s} が存在しません"

#: src/modules/displaymanager/main.py:735
msgid "Cannot configure LightDM"
msgstr "LightDMの設定ができません"

#: src/modules/displaymanager/main.py:736
msgid "No LightDM greeter installed."
msgstr "LightDM greeter がインストールされていません。"

#: src/modules/displaymanager/main.py:767
msgid "Cannot write SLIM configuration file"
msgstr "SLIMの設定ファイルに書き込みができません"

#: src/modules/displaymanager/main.py:768
msgid "SLIM config file {!s} does not exist"
msgstr "SLIM 設定ファイル {!s} が存在しません"

#: src/modules/displaymanager/main.py:894
msgid "No display managers selected for the displaymanager module."
msgstr "ディスプレイマネージャが選択されていません。"

#: src/modules/displaymanager/main.py:895
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr "globalstorage と displaymanager.conf の両方で、displaymanagers リストが空か未定義です。"

#: src/modules/displaymanager/main.py:977
msgid "Display manager configuration was incomplete"
msgstr "ディスプレイマネージャの設定が不完全です"

#: src/modules/initcpiocfg/main.py:28
msgid "Configuring mkinitcpio."
msgstr "mkinitcpioを設定しています。"

#: src/modules/initcpiocfg/main.py:203
#: src/modules/luksopenswaphookcfg/main.py:91
#: src/modules/initramfscfg/main.py:90 src/modules/openrcdmcryptcfg/main.py:77
#: src/modules/fstab/main.py:368 src/modules/localecfg/main.py:136
#: src/modules/networkcfg/main.py:40
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr " <pre>{!s}</pre> を使用するのにルートマウントポイントが与えられていません。"

#: src/modules/luksopenswaphookcfg/main.py:26
msgid "Configuring encrypted swap."
msgstr "暗号化したswapを設定しています。"

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "データのインストール。"

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "OpenRCサービスを設定"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr "ランレベル  {level!s} にサービス {name!s} が追加できません。"

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr "ランレベル  {level!s} からサービス {name!s} が削除できません。"

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""
"ランレベル {level!s} 内のサービス {name!s} に対する未知のサービスアクション <code>{arg!s}</code>。"

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr "chrootで <code>rc-update {arg!s}</code> を呼び出すとエラーコード {num!s} が返されました。"

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr "ターゲットとするランレベルは存在しません"

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr "ランレベル {level!s} のパスが <code>{path!s}</code> です。これは存在しません。"

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "ターゲットとするサービスは存在しません"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr "サービス {name!s} のパスが <code>{path!s}</code> です。これは存在しません。"

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Plymouthテーマを設定"

#: src/modules/packages/main.py:50 src/modules/packages/main.py:59
#: src/modules/packages/main.py:69
msgid "Install packages."
msgstr "パッケージのインストール"

#: src/modules/packages/main.py:57
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "パッケージを処理しています (%(count)d / %(total)d)"

#: src/modules/packages/main.py:62
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] " %(num)d パッケージをインストールしています。"

#: src/modules/packages/main.py:65
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] " %(num)d パッケージを削除しています。"

#: src/modules/bootloader/main.py:42
msgid "Install bootloader."
msgstr "ブートローダーをインストール"

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "ハードウェアクロックの設定"

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr "mkinitfsを使用してinitramfsを作成します。"

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr "ターゲットでmkinitfsを実行できませんでした"

#: src/modules/mkinitfs/main.py:50 src/modules/dracut/main.py:50
msgid "The exit code was {}"
msgstr "停止コードは {} でした"

#: src/modules/dracut/main.py:27
msgid "Creating initramfs with dracut."
msgstr "dracutとinitramfsを作成しています。"

#: src/modules/dracut/main.py:49
msgid "Failed to run dracut on the target"
msgstr "ターゲット上で dracut の実行に失敗"

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "initramfsを設定しています。"

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr "OpenRC dmcryptサービスを設定しています。"

#: src/modules/fstab/main.py:29
msgid "Writing fstab."
msgstr "fstabを書き込んでいます。"

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Dummy python job."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "Dummy python step {}"

#: src/modules/localecfg/main.py:30
msgid "Configuring locales."
msgstr "ロケールを設定しています。"

#: src/modules/networkcfg/main.py:28
msgid "Saving network configuration."
msgstr "ネットワーク設定を保存しています。"
